core-web/apps/dotcms-ui/src/app/portlets/shared/models/shared-models.ts

```diff
@@ -0,0 +1,5 @@
+export const enum LoadingState {
```

Comment:
unify this Interfase
https://github.com/dotCMS/core/blob/8848f9aeab1c050335488d5fdcc63ecc6c9660b4/core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/store/dot-palette.store.ts#L23

https://github.com/dotCMS/core/blob/585a787950fb8502edab81f84fd157f403ba4a57/core-web/apps/dotcdn/src/app/app.models.ts#L70

Reply:
I have refactored and centralized.